pkg/rtc/participant.go

```diff
@@ -697,6 +699,54 @@ func (p *ParticipantImpl) Negotiate(force bool) {
 	}
 }
 
+func (p *ParticipantImpl) clearMigrationTimer() {
+	p.lock.Lock()
+	if p.migrationTimer != nil {
+		p.migrationTimer.Stop()
+		p.migrationTimer = nil
+	}
+	p.lock.Unlock()
+}
+
+func (p *ParticipantImpl) MaybeStartMigration(force bool, onStart func()) bool {
+	if !force && p.TransportManager.HaveAllTransportEverConnected() {
```

Comment:
Making sure that all transport is connected before allowing migration. We have seen cases where migration hobbles when all transport is not fully connected.

Reply:
@boks1971  confused by the code and comment, if we don't want migrate unconnected participant, should this check be `if !force && !p.TransportManager.HaveAllTransportEverConnected` ?

Reply:
Good catch @cnderrauber . Will make a PR to fix.